lms/static/js/learner_dashboard/RecommendationsPanel.jsx

```diff
@@ -1,93 +1,76 @@
 /* global gettext */
 import React from 'react';
+import Cookies from 'js-cookie';
 
 class RecommendationsPanel extends React.Component {
   constructor(props) {
     super(props);
+    this.domainInfo = props.debug === false ? { domain: 'edx.org', expires: 365 } : {};
```

Comment:
Instead of hardcoding the domain use `SHARED_COOKIE_DOMAIN` and you won't need to add a check for debug value.

Reply:
updated.